### ISSUE TITLE
Can't run app

### ISSUE THREAD

**Issue by islamamrr** [dubnom/sandtable#6]
It's a bit unclear how to proceed after running install.sh, can you please give clearer steps as your program would be super beneficial to me

**Comment by noedger** [comment]
HiYou don't run install.sh. Give me more info on what you are trying to do. I will help you gladly. DeonSent from my Galaxy
-------- Original message --------From: islamamrr <notifications@github.com> Date: 2021/03/10  03:08  (GMT+02:00) To: dubnom/sandtable <sandtable@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [dubnom/sandtable] Can't run app (#6) 
It's a bit unclear how to proceed after running install.sh, can you please give clearer steps as your program would be super beneficial to me

**Comment by noedger** [comment]
That said. On second thought. Lets just make sure what sand table you are working on. There are a couple to chose from. Send me a link so I can give better advice. Thanks Deon. Sent from my Galaxy
-------- Original message --------From: islamamrr <notifications@github.com> Date: 2021/03/10  03:08  (GMT+02:00) To: dubnom/sandtable <sandtable@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [dubnom/sandtable] Can't run app (#6) 
It's a bit unclear how to proceed after running install.sh, can you please give clearer steps as your program would be super beneficial to me

**Comment by dubnom** [comment]
You should be able to access the interface through a web browser.  Check the port number in the config file, and the IP address/machine name of the computer you installed on.  If you still have problems, I can login remotely (If you install “tmate”).

**Comment by parkergreene** [comment]
I too and having some difficulty getting the program up and running.  I cloned the GitHub repo on a raspberry pi 3b+ and customized the config file according to my machine information.  I'm unclear as to what the next step is to start the server on the pi though.  I tried just running my config python file but that didn't seem to work (tried http:localhost on the pi and a separate computer with the Pi's up address, neither would connect).  Could you clarify what the startup process is for the server?  Thanks!

**Comment by dubnom** [comment]
Call me tomorrow (Monday), New York time at 917-825-5551.
I’m pretty much free from 10:00am on.
- Mike

Sent from my iPad

On Apr 10, 2022, at 11:30 PM, pgreene16 ***@***.***> wrote:

﻿

I too and having some difficulty getting the program up and running. I cloned the GitHub repo on a raspberry pi 3b+ and customized the config file according to my machine information. I'm unclear as to what the next step is to start the server on the pi though. I tried just running my config python file but that didn't seem to work (tried http:localhost on the pi and a separate computer with the Pi's up address, neither would connect). Could you clarify what the startup process is for the server? Thanks!

**Comment by dubnom** [comment]
Also, did you run install.sh?

Sent from my iPad

On Apr 10, 2022, at 11:30 PM, pgreene16 ***@***.***> wrote:

﻿

I too and having some difficulty getting the program up and running. I cloned the GitHub repo on a raspberry pi 3b+ and customized the config file according to my machine information. I'm unclear as to what the next step is to start the server on the pi though. I tried just running my config python file but that didn't seem to work (tried http:localhost on the pi and a separate computer with the Pi's up address, neither would connect). Could you clarify what the startup process is for the server? Thanks!

**Comment by parkergreene** [comment]
Yes.  I'm new to Linux but was finally able to figure out how to run "sudo bash install sh".  I saw all of the libraries install.  I rebooted afterwards but still no luck connecting via localhost of the ip address (I assume I need to run another script to start the server).

**Comment by dubnom** [comment]
The install script should have placed an automatic startup of the server into /etc/rc.local
Linux log files are stored in /var/log and the sandtable stores its logs there.
There is a program called ‘tmate’ which would allow you to give me access to the machine so I can see what is going on.
sudo app install tmate
              tmate

then send me the key so I can attach and take a look.

  *   Mike

**Comment by hackintoshlover** [comment]
> Yes. I'm new to Linux but was finally able to figure out how to run "sudo bash install sh". I saw all of the libraries install. I rebooted afterwards but still no luck connecting via localhost of the ip address (I assume I need to run another script to start the server).

Hi,
Do the following:
1) Remove the script from rc.local
Type sudo nano /etc/rc.local in a terminal
delete the line that says /var/www/sandtable/bin/rc.sandtable
ctrl+x then y
2) sudo rm -rf /var/www
3) sudo rm -rf /home/sandtable
4) sudo deluser -remove-home sandtable
5) Considering you have cloned the sandtable folder in the pi directory
Edit the install.sh script located in /sandtable/bin
Edit line 
line 43 ln -s $PWD /var/www/    to    ln -s /home/pi/sandtable /var/www/
line 44 ln -s $PWD /home/sandtable/       to       ln -s /home/pi/sandtable /home/sandtable/
ctrl+s
6) Edit the config file and change port to 3000
ctrl+s
7) Run sudo bash install.sh in the bin directory
8) Upon completion reboot
9) To connect type http://ip of your raspberry pi:3000 in an alternate browser and your website should be visible

Hope that helps or you must have figured it out earlier.
